src/ocean/util/container/cache/model/containers/KeyToNode.d

```diff
@@ -85,22 +85,37 @@ class KeyToNode: HashMap!(TimeToIndex.Node*)
 
     public this ( size_t n )
     {
-        super(this.allocator = new ArrayAllocatedFreeBucketElements(n), n);
+        this.allocator = new ArrayAllocatedFreeBucketElements(n);
+        super(this.allocator, n);
```

Comment:
This seems to be a refactoring, nothing to do with the fix.

Reply:
yes, it's a refactoring